index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header class="main-header">
+
+            <nav class="header">
+
+                <div class="name"><h1>Paul Orife</h1></div>
```

Comment:
- [ ] You did a great job working on the toolbar of your project! 💯 However, as the previous reviewer requested, this is supposed to be a "logo", so it would be better if instead of wrapping it inside an `<h1>` tag you used an `<a>` tag. This is because, by convention, logos are used to navigate back to home pages, so including this behavior would improve tremendously user experience! 🤓 🛠️  

```suggestion
                <div class="name"><a href="#">Paul Orife</a></div>
```

Reply:
> * [ ]  You did a great job working on the toolbar of your project! 💯 However, as the previous reviewer requested, this is supposed to be a "logo", so it would be better if instead of wrapping it inside an `<h1>` tag you used an `<a>` tag. This is because, by convention, logos are used to navigate back to home pages, so including this behavior would improve tremendously user experience! 🤓 🛠️

Thank you for the guideline, i have made adjustment and i have submitted the pull request link.